plugins/robots/interpreters/nxtKitInterpreter/src/nxtAdditionalPreferences.cpp

```diff
@@ -72,7 +102,11 @@ void NxtAdditionalPreferences::restoreSettings()
 
 void NxtAdditionalPreferences::onRobotModelChanged(kitBase::robotModel::RobotModelInterface * const robotModel)
 {
+	QLOG_DEBUG() << robotModel->name();
 	mUi->bluetoothSettingsGroupBox->setVisible(robotModel->name() == mBluetoothRobotName);
+	if (QOperatingSystemVersion::currentType() == QOperatingSystemVersion::OSType::Unknown) {
```

Comment:
`Unknown` means that `QOperatingSystemVersion` does not have a special literal name for a variant. I suggest using `PlatformInfo::osType` here